q2_feature_classifier/_vsearch.py

```diff
@@ -25,53 +30,159 @@ def classify_consensus_vsearch(query: DNAFASTAFormat,
                                min_consensus: float = 0.51,
                                unassignable_label: str =
                                _get_default_unassignable_label(),
+                               search_exact: bool = False,
+                               top_hits_only: bool = False,
                                threads: str = 1) -> pd.DataFrame:
     seqs_fp = str(query)
     ref_fp = str(reference_reads)
     cmd = ['vsearch', '--usearch_global', seqs_fp, '--id', str(perc_identity),
            '--query_cov', str(query_cov), '--strand', strand, '--maxaccepts',
            str(maxaccepts), '--maxrejects', '0', '--output_no_hits', '--db',
-           ref_fp, '--threads', str(threads), '--blast6out']
+           ref_fp, '--threads', str(threads)]
+    if search_exact:
+        cmd[1] = '--search_exact'
+    if top_hits_only:
+        cmd.append('--top_hits_only')
+    cmd.append('--blast6out')
     consensus = _consensus_assignments(
         cmd, reference_taxonomy, min_consensus=min_consensus,
         unassignable_label=unassignable_label)
     return consensus
 
 
+def classify_hybrid_vsearch_sklearn(ctx,
```

Comment:
A general note - should we start publishing the intermediate `FeatureData[Sequence]` & `FeatureData[Taxonomy]` created as part of our classifier training? Those Artifacts were helpful for me to have when testing this new Pipeline.

Reply:
A second general note - I ran this on MP tutorial and had 100% of the classification performed by sklearn. Should I look at another tutorial data set, or would you expect the same there?

Reply:
intermediate files == ref sequences? Would not hurt to publish (though then we get back into the issue of database devs not wanting us to re-host their data). If the classifier training tutorial were on the 99% OTUs instead of the 88%, we would have intermediate files and classifier trained with the current sklearn release every release... but then it bogs down busywork. I am ranting now.

> I ran this on MP tutorial and had 100% of the classification performed by sklearn. Should I look at another tutorial data set, or would you expect the same there?
Maybe try out on another dataset? On some of my own data I was finding majority VSEARCH classification... it may just depend on the dataset.